client/platform/desktop/backend/native/common.ts

```diff
@@ -816,45 +816,47 @@ async function _importTrackFile(
 ) {
   /* Look for JSON track file as first priority */
   let foundDetections = false;
-  const foundTrackFileAbsPath = await _findJsonTrackFile(jsonMeta.originalBasePath);
-  const trackFileAbsPath = userTrackFileAbsPath !== undefined
-    ? userTrackFileAbsPath : foundTrackFileAbsPath;
-  if (trackFileAbsPath !== null && !CsvFileName.test(trackFileAbsPath)) {
+  if (userTrackFileAbsPath !== null) {
+    const foundTrackFileAbsPath = await _findJsonTrackFile(jsonMeta.originalBasePath);
+    const trackFileAbsPath = userTrackFileAbsPath !== undefined
+      ? userTrackFileAbsPath : foundTrackFileAbsPath;
```

Comment:
`userTrackFileAbsPath` is annotated in the stub as `string | null` and then compared to `undefined` inside a conditional block where it's already been established as `not null`.  

I'm going to venture a guess that you probably don't need to call `_findJsonTrackFile` at all in this function because it was called already in the pre-import step.  

I'm going to venture a guess that because `foundDetections` requires `trackFileAbsPath` requires `userTrackFileAbsPath` which is always set inside the new conditional you added, that variable is doing no work at all and the block inside `Look for other types of annotation files as a second priority` is now unreachable code.  Is that right?

Reply:
Yeah, I think we need to get rid of the test on line 823 for `trackFileAbsPath` so it's just "If it's a json file do this, otherwise if it's a CSV file do this".

The comment about look ing for other types of annotation files is misleading.  there should no longer be any file discover going on in this function.  It's just handling the value of `userTrackFileAbsPath`

Reply:
For the case of not importing stereoscopic data that is correct.  In stereoscopic there is no precursor to finding the track files for each camera.  That should probably be done as another PR where you define the folder and assign track files for each camera if needed.  
If you import stereoscopic right now `userTrackFileAbsPath` is `undefined` and it should look for other annotation files associated with each camera.  It does import those relative files but right now the display doesn't show them.  Maybe this update should be apart of my stereoscopic viewing update.

Reply:
I see, thanks.